### DISCUSSION TITLE
"-undothrow 2" crashing compilation sessions

### DISCUSSION THREAD

**Discussion by lievendf** [vscode-abl/vscode-abl#242] [Q&A]
This is probably not related to the VSCode plugin, but I've just recently added the "-undothrow 2" startup parameter on my development environment (to uncover errors that might have gone under the radar because of missing explicit error handling), and found all my compilation sessions crashing after a while (and not returning, as they all crash in the same 10 second interval I assume).
Has anyone else used this parameter and/or seen similar issues?

More info: https://docs.progress.com/bundle/abl-reference/page/BLOCK-LEVEL-ON-ERROR-UNDO-THROW-statement.html

**Comment by gquerret** [comment]
Maybe you could cross-post that on Communities, as the option may have an impact also on PDSOE sessions.

**Reply by lievendf** [reply]
I've not been able to reproduce this in PDSOE yet...

**Comment by gquerret** [comment]
Converting to discussion as this is not an issue per se.
